app/src/main/java/com/github/sdpteam15/polyevents/helper/GoogleMapHelper.kt

```diff
@@ -426,4 +432,42 @@ object GoogleMapHelper {
         }
         tempPoly?.points = tempLatLng
     }
+
+
+    fun areasToFormattedStringLocations(
```

Comment:
Is there a reason why the coordinates are stored as String and not e.g. as GeoPoint (supported by Firebase) ?
If it works like that it's good but I'm just curious :)

Reply:
Honnestly, I didn't see the GeoPoint type supported by Firebase :)
I just serialised the coordinates into one string :)